iothub/service/src/Amqp/AmqpConnectionHandler.cs

```diff
@@ -121,19 +112,19 @@ internal async Task OpenAsync(CancellationToken cancellationToken)
                         CertificateValidationCallback = _options.RemoteCertificateValidationCallback,
                     };
 
-                    amqpTransportInitiator = new AmqpTransportInitiator(amqpSettings, tlsTranpsortSettings);
-                }
+                    var amqpTransportInitiator = new AmqpTransportInitiator(amqpSettings, tlsTranpsortSettings);
 
-                try
-                {
-                    _transport = await amqpTransportInitiator.ConnectAsync(cancellationToken).ConfigureAwait(false);
-                }
-                catch (Exception ex) when (ex is not AuthenticationException)
-                {
-                    if (Logging.IsEnabled)
-                        Logging.Error(this, ex, nameof(OpenAsync));
+                    try
+                    {
+                        _transport = await amqpTransportInitiator.ConnectAsync(cancellationToken).ConfigureAwait(false);
+                    }
+                    catch (Exception ex) when (ex is not AuthenticationException)
+                    {
+                        if (Logging.IsEnabled)
+                            Logging.Error(this, ex, nameof(OpenAsync));
 
-                    throw;
+                        throw;
```

Comment:
Will AuthenticationException be bubbled up to the user or wrapped in an IotHubServiceException?

Reply:
well any of the other types